src/Tasks/Microsoft.NET.Build.Tasks/ConflictResolution/RemoveDepsFileConflicts.cs

```diff
@@ -0,0 +1,156 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Microsoft.Extensions.DependencyModel;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Microsoft.NET.Build.Tasks.ConflictResolution
+{
+    public partial class RemoveDepsFileConflicts : Task
```

Comment:
I'm not sure if this was in your plan or not, but instead of generating everything into the .deps.json file, and then removing the conflicts in a separate task, we should just be feeding the Conflicts into the original GenerateDepsFile task.  Then we can generate the whole thing in one shot.

It makes the logic simpler, and more efficient.